File: NormalLogAllPairsOfArray.java
class NormalLogAllPairsOfArray {
  public static void main(String[] args) {

  int boxes[] = {1,2,3,4,5};
  for(int a=0; a<boxes.length; a++){
    for(int b=0; b<boxes.length; b++ ){
      System.out.println(boxes[a] + " , " + boxes[b]);
    }
  }
}
}

File: NormalLogAllPairsOfArray.js
//log all pairs of an array
const boxes = [1,2,3,4,5]

function NormalLogAllPairsOfArray(array){
  for(let i=0; i<array.length; i++){
    for(let x=0; x<array.length; x++){
      console.log(array[i],array[x]);
    }
  }
}

NormalLogAllPairsOfArray(boxes);
